instagram/src/components/Login/LoginPage.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import "./login.css";
+// updateState={this.props.updateState}
+
```

Comment:
loving the authentication stretch and how well you implemented it with the React Lifestyle functions below. Your grasp and understand of React has truly been growing and your hardwork and dedication shows. This is portfolio level quality work.